examples/pre-67u3/controlplane/controlplane.yaml

```diff
@@ -21,9 +21,6 @@ apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
 kind: VSphereMachine
 metadata:
   name: ${CLUSTER_NAME}-controlplane-0
-  labels:
```

Comment:
Hi @andrewsykim,

Please add this back. @yastij fixed this in https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/556 and it's consistent with CAPA.